source/extensions/config_subscription/grpc/grpc_mux_failover.h

```diff
@@ -6,6 +6,8 @@
 namespace Envoy {
 namespace Config {
 
+class GrpcMuxFailoverTest;
```

Comment:
is forward declaration really needed here?

Reply:
No it is not. Removed.
Thanks for the great comments!